.github/workflows/release-suite-native-preview-update.yml

```diff
@@ -0,0 +1,49 @@
+name: "[Release] suite-native preview update"
+
+on:
+  push:
+    branches: [develop]
+  pull_request:
+    paths:
+      - "suite-native/**"
+      - "suite-common/**"
+  workflow_dispatch:
+
+  jobs:
+    update:
+      if: github.repository == 'trezor/trezor-suite' || github.repository == 'trezor/trezor-suite-private'
+    name: EAS Update
+    environment: develop-suite-native
+    runs-on: ubuntu-latest
+    concurrency: fingerprint-${{ github.workflow }}-${{ github.head_ref || github.run_id }}
```

Comment:
What about adding a `cancel-in-progress` argument to concurrency runs? So we can save some resources? Just an idea. Quite often, someone is pushed to the PR a few times in a short period. This parameter would ensure that the job won't run for every push.

Reply:
agree